docs/events/timestamps/index.md

```diff
@@ -0,0 +1,33 @@
+| `dvce_created_tstamp` | [Tracker](/docs/sources/trackers/index.md)                                          | The device's timestamp when the event was created                                                                                                                      | ✅             |
+| `dvce_sent_tstamp`    | [Tracker](/docs/sources/trackers/index.md)                                          | The device's timestamp when the event was successfully sent to the Collector endpoint                                                                                  | ✅             |
+| `true_tstamp`         | You                                                                                 | Exact timestamp, defined within your tracking code                                                                                                                     | ❌             |
+| `collector_tstamp`    | [Collector](/docs/pipeline/collector/index.md)                                      | When the Collector received the event payload                                                                                                                          | ✅             |
+| `derived_tstamp`      | [Enrich](/docs/api-reference/enrichment-components/index.md)                        | Calculated from other timestamps, or the same as `true_tstamp`                                                                                                         | ✅             |
+| `etl_tstamp`          | [Enrich](/docs/api-reference/enrichment-components/index.md)                        | The time at which Enrich started processing the event                                                                                                                  | ✅             |
+| `refr_device_tstamp`  | [Enrich](/docs/api-reference/enrichment-components/index.md)                        | Timestamp extracted from the [cross-domain navigation query string](/docs/pipeline/enrichments/available-enrichments/cross-navigation-enrichment/index.md), if present | ❌             |
+| `load_tstamp`         | [Loader](/docs/destinations/warehouses-lakes/loading-process/index.md) or warehouse | Timestamp when the event was loaded into the warehouse                                                                                                                 | ✅             |
+
+The `load_tstamp` is added either by the Loader or by the warehouse at the point of loading, depending on the Loader/warehouse. Use this timestamp for incremental processing.
+
+## Derived timestamp
+
+Snowplow recommends using the `derived_tstamp` as the primary event timestamp for analysis. It allows for devices with incorrectly set clocks, or delays in event sending due to network outages.
```

Comment:
This might be true (not my area!) but a conflicting point is that downstream analysis should pay attention to how the warehouse table is partitioned.

For example, if the warehouse is partitioned by `load_tstamp`, then it is really important to use clauses like `WHERE load_tstamp > ?`.  I don't think we should give the impression that `derived_tstamp` is the one and only timetsamp to use in analysis.